test/test_formatters.rb

```diff
@@ -63,6 +63,27 @@ def test_vim
     ))
   end
 
+  def test_vim_with_line_numbers
+    vim = formatter_for(:format => 'vim', :line_numbers => true)
+    assert_equal %{C\t./script.rb\t/^class C < D$/;"\tc\tline:1\tclass:A.B\tinherits:D}, vim.format(build_tag(
+      :kind => 'class', :name => 'C',
+      :pattern => "class C < D",
+      :class => 'A::B', :inherits => 'D'
```

Comment:
None if these tests needs either `:class` nor `:inherits` since those are not the features that we're interested here. You can take those parameters out.